Modules/Files/test/Files.API.Tests.Performance.EndToEnd/src/download-file.test.ts

```diff
@@ -0,0 +1,77 @@
+import http from "k6/http";
+import { Options } from "k6/options";
+import {
+  describe,
+  expect,
+} from "https://jslib.k6.io/k6chaijs/4.3.4.2/index.js";
+import { getJwt, assertEnvVarExists, Size, tomorrow } from "./utils";
+
+assertEnvVarExists();
```

Comment:
I actually wanted you to have a method to which you pass the name of an env var, which checks if an env var with this name exists.

Since all the tests need the same env vars, it's maybe actually a good idea to have a method like your current implementation as well. But this method should have a more appropriate name then (something like `assertAllRequiredEnvVarsExist`).

Or maybe even better: a method called `getConfiguration()`, which validates that all required env vars exist, reads them and returns them in a strongly typed object.

Reply:
After thinking about it for a bit, I went for the getConfiguration() method implementation. For the way we currently have the tests setup (including the run-all scripts), I think it makes sense to have the validations done all at once, and return a configuration object with the actual "options", I don't think we need a function that validates each env var separately, at least for now.